packages/lesswrong/lib/collections/users/custom_fields.ts

```diff
@@ -991,7 +991,7 @@ addFieldsDict(Users, {
     canCreate: ['members'],
     canUpdate: [userOwns, 'sunshineRegiment', 'admins'],
     group: forumTypeSetting.get() === "EAForum" ? formGroups.aboutMe : formGroups.siteCustomizations,
-    order: forumTypeSetting.get() === "EAForum" ? 9 : 101,
+    order: forumTypeSetting.get() === "EAForum" ? 5 : 101,
```

Comment:
What does this change?

Reply:
It changes the order that the fields show up in the form - I did a bit of reordering for the Edit Profile form fields to better match the profile page layout.